ads/_config.js

```diff
@@ -15,235 +15,313 @@
  */
 
 /**
- * URLs to prefetch for a given ad type.
+ * The config of each ad network.
+ * Please keep the list alphabetic order.
  *
- * This MUST be kept in sync with actual implementation.
+ * yourNetworkName: {  // This is the "type" attribute of <amp-ad>
```

Comment:
I'm hesitating. This free form gives me space to insert descriptions for each field. And meantime, I don't think we're doing type checks on 3p code anyway.

So maybe leave it as is now, and later if we want to do type-check for 3p code, then change them together.